danielortizlira/backstage
# Add backstage internal components overrides example

**_This PR is for demonstration purposes only._**

# How to test

This PR [is based on the changes that add support](https://github.com/danielortizlira/backstage/pull/1) to `material-ui` overrides in Backstage internal components. In order to see it work the example app should use the local `@backstage/core-components` package, for that follow these instructions:

- `cd packages/core-components`
- `yarn build & yarn link`
- `cd ../../ & yarn link @backstage/core-components`
- `yarn dev`
